src/dataexplorer/dataexplorer-application/src/components/Navigation/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -7,44 +7,26 @@ import {
     FileTextOutlined,
 } from '@ant-design/icons';
 import './Breadcrumbs.style.scss';
+import { BreadcrumbContext } from './BreadcrumbContext';
 
-interface IBreadCrumbsProps {
-    setName: (name: string) => void;
-    setSchemaNumber: (id: string) => void;
-    name: string;
-    schemaNumber: string;
-}
-
-const Breadcrumbs = ({
-    setName,
-    setSchemaNumber,
-    name,
-    schemaNumber,
-}: IBreadCrumbsProps) => {
-    function fromSchemeToHome() {
-        setSchemaNumber('');
-        setName('');
-    }
-    function fromSchemeToPatient() {
-        setSchemaNumber('');
-    }
+const Breadcrumbs = () => {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    const { name, schemaNumber } = useContext(BreadcrumbContext);
     return (
         <>
             <div className="breadcrumbs-container">
                 <Breadcrumb>
                     <Breadcrumb.Item>
-                        <Link to="/" onClick={() => fromSchemeToHome()}>
-                            <HomeOutlined className="bread-icon" />
+                        <Link to="/">
+                            <HomeOutlined />
```

Comment:
Any reason why the styling class is removed?

Reply:
That is an icon - icons are (will be) removed, so it doesn't matter.

Reply:
This happened during a rebase, and was not intended. Since this will be removed anyway, I suggest to let it stay removed. Do you agree?